prometheus-metrics-ghc/src/Prometheus/Metric/GHC.hs

```diff
@@ -125,13 +134,193 @@ ghcCollectors = [
             parMaxBytesCopied
     ]
 
+#else
+
+ghcCollectors :: [IO [SampleGroup]]
+ghcCollectors = [
+      statsCollector
+            "ghc_gcs_total"
+            "Total number of GCs"
+            CounterType
+            gcs
+    , statsCollector
+            "ghc_major_gcs_total"
+            "Total number of major (oldest generation) GCs"
+            CounterType
+            major_gcs
+    , statsCollector
+            "ghc_allocated_bytes_total"
+            "Total bytes allocated"
+            CounterType
+            allocated_bytes
+    , statsCollector
+            "ghc_max_live_bytes"
+            "Maximum live data (including large objects + compact regions)"
+            CounterType -- TODO: Shouldn't it be a Gauge? It only ever goes up, but it's not counting anything
```

Comment:
My hunch is Gauge for this and the rest of the max_foo_bytes things, but I don't know for sure. I've asked on #prometheus and will get back to you if I get an answer.

Reply:
```
<bbrazil> gauge
<bbrazil> you care about the absolute value
```

Thus I'd suggest changing all the following to gauges:
- `ghc_max_live_bytes`
- `ghc_max_large_object_bytes`
- `ghc_max_compact_bytes`
- `ghc_max_slop_bytes`
- `ghc_max_mem_in_use_bytes`

Reply:
Thanks for obtaining this info, I'll change these to gauges.